Common.mk

```diff
@@ -221,7 +221,7 @@ trimap = $(and $(strip $2),$(strip $3),$(strip $4),$(call \
 
 _pos = $(if $(filter $1,$2),$(call _pos,$1,\
        $(call list-rem,$2),x $3),$3)
-pos = $(words $(call _pos,$1,$2))
+pos = $(words $(call _pos,$1,$2,))
```

Comment:
this was just a long standing bug that only shows it self in a couple builds, and even then its just a warning but its confusing so im fixing it.